spec/models/user_model_spec.rb

```diff
@@ -0,0 +1,118 @@
+require "rails_helper"
+
+
+RSpec.describe User, type: :model do
+  before(:each) do
+    @user = FactoryGirl.create(:user)
+    @vacation = FactoryGirl.build(:vacation, start_date: Date.today, end_date: Date.today + 2.days, user_id: @user.id)
+  end
+
+  describe "Associations" do
+    it { should have_many(:timelogs) }
+    it { should have_many(:identities) }
+    it { should have_many(:vacations) }
+    it { should have_many(:salaries) }
+    it { should have_many(:user_skills) }
+    it { should have_many(:skills) }
+  end
+
+  describe "correct roles" do
+    it "assigns right roles" do
+      expect(User.roles["admin"]).to eql(0)
+      expect(User.roles["member"]).to eql(1)
+    end
+  end
+
+  describe "correct statuses" do
+    it "assigns right statuses" do
+      expect(User.statuses["inactive"]).to eql(0)
+      expect(User.statuses["active"]).to eql(1)
+    end
+  end
+
+  describe "sick_leave_approved" do
```

Comment:
1. It is better to put these tests in context `ex: "[vacation counters]"`.
2. Please try to use `let` method, instead of using `@var` https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let